app/history_v2_test.go

```diff
@@ -78,6 +78,25 @@ func TestHistoryV2Handler(t *testing.T) {
 				g.Assert(len(messages)).Equal(0)
 				Expect(err).To(BeNil())
 			})
+
+			g.It("It should return 200 and [] if the user is authorized into the topic but there are only blocked messages", func() {
```

Comment:
Please, can we add a similar test on `history_test.go`? By doing that we guarantee that both endpoints will not send blocked messages.